src/com/jalasoft/search/common/Validator.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Validator .java
+ *
+ * Copyright (c) 2018 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information").
+ * You shall not disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into with Jalasoft.
+ */
+package com.jalasoft.search.common;
+
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Paths;
+
+/*
+* Class to manage validations related to files
+* @version  1.0
+* @author Eliana Navia
+*/
+public class Validator {
```

Comment:
this class only going to validate files, you can rename to other name with more especifications e.g.: FileValidator